docs/source/distributed.optim.md

```diff
@@ -1,11 +1,15 @@
+```{eval-rst}
 .. role:: hidden
     :class: hidden-section
+```
 
-Distributed Optimizers
-======================
+# Distributed Optimizers
 
-.. warning ::
-    Distributed optimizer is not currently supported when using CUDA tensors
+:::{warning}
+Distributed optimizer is not currently supported when using CUDA tensors
+:::
 
+```{eval-rst}
```

Comment:
The build [issue](https://github.com/pytorch/pytorch/actions/runs/15576838598/job/43911764169?pr=155528#step:11:828) is in the docstring itself. Need to change this [link](https://github.com/pytorch/pytorch/blob/458cc7213baf01ab742a2250c7a31f7796fcb1ef/torch/distributed/optim/zero_redundancy_optimizer.py#L287) to 
```
 `ZeRO <https://arxiv.org/abs/1910.02054>`_
 ```
 and delete this [line](https://github.com/pytorch/pytorch/blob/458cc7213baf01ab742a2250c7a31f7796fcb1ef/torch/distributed/optim/zero_redundancy_optimizer.py#L369C8-L369C47). 
 Need to check this page in the preview build: 
 https://docs.pytorch.org/docs/main/distributed.optim.html#torch.distributed.optim.ZeroRedundancyOptimizer